types/node-telegram-bot-api/index.d.ts

```diff
@@ -1634,22 +1634,28 @@ declare class TelegramBot extends EventEmitter<
 
     createChatInviteLink(
         chatId: TelegramBot.ChatId,
-        name?: string,
-        expire_date?: number,
-        member_limit?: number,
-        creates_join_request?: boolean,
+        options?: {
+            name?: string,
+            expire_date?: number,
+            member_limit?: number,
+            creates_join_request?: boolean
+        }
     ): Promise<TelegramBot.ChatInviteLink>;
 
     editChatInviteLink(
         chatId: TelegramBot.ChatId,
         inviteLink: string,
-        name?: string,
-        expire_date?: number,
-        member_limit?: number,
-        creates_join_request?: boolean,
+        options?: {
+            name?: string,
+            expire_date?: number,
+            member_limit?: number,
+            creates_join_request?: boolean
+        }
     ): Promise<TelegramBot.ChatInviteLink>;
 
-    revokeChatInviteLink(chatId: TelegramBot.ChatId, inviteLink: string): Promise<TelegramBot.ChatInviteLink>;
+    revokeChatInviteLink(chatId: TelegramBot.ChatId, options?: {
```

Comment:
This type definition does not match [the actual code](https://github.com/yagop/node-telegram-bot-api/blob/595cdbd0c656be258bebda2aab5d343de1e938d4/src/telegram.js#L1616), which takes `inviteLink: string` as a second argument, and accepts an optional third argument. I suspect that the original doc (probably generated from function comments at <https://github.com/yagop/node-telegram-bot-api/blob/595cdbd0c656be258bebda2aab5d343de1e938d4/src/telegram.js#L1611-L1612>) is not correct.

Would you like to update this PR to fix it? Also, it might be a good option to submit an additional PR to the source repo to fix the documentation.

Reply:
I will update to fix it! Thanks for checking that.